checkClick.js

```diff
@@ -0,0 +1,16 @@
+import { gameScore } from './script.js'
+
+export let score = 0
+
+export const checkClick = ({target}) => {
+    if (target.className === 'box__circle red') {
```

Comment:
correct way to check if given element has class is:
```
target.className.contains('...')
```

because your way won't work for element with multiple clases